### PR TITLE
Issue 18 - Utility 5 - Implement mobile buttons

### PR DESCRIPTION
**Pull request by FrancescoSTL**
This PR adds the html buttons for utility sidebar 5- mobile.

Issue ref: #18

### CODE REVIEW

**Comment by FrancescoSTL** [comment]
r? 👍

File: data/tmpl/utility/content5.html [hunk 1]
```diff
@@ -13,7 +13,8 @@
       <main>
         <h1>Free to Roam</h1>
         <p>Take your history, bookmarks and passwords with you.</p>
-        <button class="button">Get Firefox on your phone.</button>
+        <button class="button mobile" target="_blank" onclick="window.open('https://itunes.apple.com/us/app/apple-store/id989804926?pt=373246&mt=8&ct=firefox-onboarding-addon', '_blank')">Get Firefox for iOS.</button>
```
Comment (schalkneethling) [inline data/tmpl/utility/content5.html]: @chrismore Should we not be using the official badges here for the Play and App store respectively?

Comment (schalkneethling) [inline data/tmpl/utility/content5.html]: I would grab the badges from here:
https://github.com/mozilla/bedrock/tree/master/media/img/l10n/en-US/firefox/android
and here
https://github.com/mozilla/bedrock/tree/master/media/img/l10n/en-US/firefox/ios

They are the official latest badges and have already been optimized.

**Comment by FrancescoSTL** [comment]
r? :neckbeard:

File: data/css/all-aboard.css [hunk 2]
```diff
@@ -205,6 +205,10 @@ header {
 	transition: background-color .1s ease-in-out;
 }
 
+.mobile img{
```
Comment (schalkneethling) [inline data/css/all-aboard.css]: Nit: Space after `img`

File: data/css/all-aboard.css [hunk 3]
```diff
@@ -205,6 +205,10 @@ header {
...
+    margin: 10px 0px 10px 0px;
```
Comment (schalkneethling) [inline data/css/all-aboard.css]: No need for `px` after `0` also, you can shorten this to just `margin: 10px 0;`

File: data/media/appleStore.svg [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+			<path fill="#FFFFFF" d="M121.207,10.853c0,0.188-0.014,0.346-0.039,0.475h-3.143c0.014,0.466,0.164,0.821,0.455,1.067
+				c0.266,0.22,0.609,0.33,1.029,0.33c0.465,0,0.889-0.074,1.271-0.223l0.164,0.728c-0.447,0.194-0.973,0.291-1.582,0.291
+				c-0.73,0-1.305-0.215-1.721-0.645c-0.418-0.43-0.625-1.007-0.625-1.731c0-0.711,0.193-1.303,0.582-1.775
+				c0.406-0.504,0.955-0.756,1.648-0.756c0.678,0,1.193,0.252,1.541,0.756C121.068,9.77,121.207,10.265,121.207,10.853z
+				 M120.207,10.582c0.008-0.311-0.061-0.579-0.203-0.805c-0.182-0.291-0.459-0.437-0.834-0.437c-0.342,0-0.621,0.142-0.834,0.427
+				c-0.174,0.227-0.277,0.498-0.311,0.815H120.207z"/>
+		</g>
+	</g>
+</g>
+</svg>
```
Comment (schalkneethling) [inline data/media/appleStore.svg]: Oh, how interesting. This can definitely do with an optimization round using svgomg

File: data/tmpl/utility/content5.html [hunk 5]
```diff
@@ -13,7 +13,12 @@
       <main>
         <h1>Free to Roam</h1>
         <p>Take your history, bookmarks and passwords with you.</p>
-        <button class="button">Get Firefox on your phone.</button>
+        <a class="mobile" href="https://itunes.apple.com/us/app/apple-store/id989804926?pt=373246&mt=8&ct=firefox-onboarding-addon" target="_blank">
```
Comment (schalkneethling) [inline data/tmpl/utility/content5.html]: escape the `&` with '&amp;`

File: data/tmpl/utility/content5.html [hunk 6]
```diff
@@ -13,7 +13,12 @@
...
+          <img src="/data/media/appleStore.svg" />
+        </a>
+        <a class="mobile" href="https://play.google.com/store/apps/details?id=org.mozilla.firefox&referrer=utm_source=firefox-browser&utm_medium=Referral&utm_campaign=onboarding-addon" target="_blank">
+          <img src="/data/media/playStore.png" />
```
Comment (schalkneethling) [inline data/tmpl/utility/content5.html]: For this, and the image before, specify at least the width and provide and `alt`

**Comment by FrancescoSTL** [comment]
Third time's a charm. @schalkneethling r?

**Comment by schalkneethling** [comment]
I am totally OK with the links to the app stores opening in a new window, just want to double check with @chrismore that this is what was intended

**Comment by FrancescoSTL** [comment]
@schalkneethling we discussed yesterday and since we can't explicitly open in new tab (_blank link window vs tab action is set by the user in preferences), we decided to leave it for now and then review the experience in London when it is all built

**Comment by schalkneethling** [comment]
"we can't explicitly open in new tab " - Technically we can, but we have to send an intent to the add-on and then use `tabs.open `~ https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/tabs#open(options)

I am going to merge this as is as it functionally is sound but, we can rethink this along with Matt.
